### PR TITLE
Fixed: Misplaced contents in Search Fragment #931

### LINKED ISSUE #931: Content in the Search is mispositioned in landscape orientation
**Describe the bug**
The content is mispositioned in the Search fragment in landscape orientation

**To Reproduce**
1. Open Search fragment in the landscape mode. 

**Expected behavior**
1. To position it in the middle

**Screenshots**

Emulator : 
screenshot_1547977987

Device:
hello

**Smartphone Info:**

| |               | |
|-|---------------|-|
|1|Device         |Emulator Pixel 2|
||Android Version| Android 9 |
|2|Device         |Samsung J3 |
||Android Version| Android 5.1 |

**Would you like to work on the issue?**

 Yes

### PR DESCRIPTION
**Pull request by RebelsterPi**
Fixes #931 

Changes: Created a landscape variation

Screenshots for the change:
screenshot_1548574345
(Pixel 2 Emulator)

screenshot_2019-01-27-13-03-08-1
(Samsung J3 device)

### CODE REVIEW

File: app/src/main/res/layout-land/fragment_search.xml [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+                    android:layout_alignEnd="@id/timeTextView"
+                    android:layout_alignLeft="@+id/timeTextView"
+                    android:layout_alignRight="@id/timeTextView"
+                    android:layout_alignStart="@+id/timeTextView"
+                    android:layout_below="@id/timeTextView"
+                    android:layout_gravity="center_horizontal"
+                    android:background="@color/grey" />
+            </RelativeLayout>
+
+
```
Comment (nikit19) [inline app/src/main/res/layout-land/fragment_search.xml]: extra

File: app/src/main/res/layout-land/fragment_search.xml [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+            <TextView
+                android:layout_width="wrap_content"
+                android:layout_height="wrap_content"
+                android:layout_marginTop="@dimen/layout_margin_large"
+                android:text="@string/in"
+                android:textSize="@dimen/text_size_large" />
+
+
```
Comment (nikit19) [inline app/src/main/res/layout-land/fragment_search.xml]: extra

File: app/src/main/res/layout-land/fragment_search.xml [hunk 3]
```diff
@@ -0,0 +1,102 @@
...
+                    android:layout_height="@dimen/event_name_divider_height"
+                    android:layout_alignEnd="@id/locationTextView"
+                    android:layout_alignLeft="@+id/locationTextView"
+                    android:layout_alignRight="@id/locationTextView"
+                    android:layout_alignStart="@+id/locationTextView"
+                    android:layout_below="@id/locationTextView"
+                    android:layout_gravity="center_horizontal"
+                    android:background="@color/grey" />
+            </RelativeLayout>
+
```
Comment (GOVINDDIXIT) [inline app/src/main/res/layout-land/fragment_search.xml]: This also

Comment (RebelsterPi) [reply app/src/main/res/layout-land/fragment_search.xml]: removed the extra line

File: app/src/main/res/layout-land/fragment_search.xml (hunk 2 above)
Comment (RebelsterPi) [reply app/src/main/res/layout-land/fragment_search.xml]: I have made the the changes

**Comment by iamareebjamal** [comment]
I dont agree with adding a landscape layout just to fix search view. We cannot maintain this faithfully

Either think of a different solution or just extract the sub-layout which is different in both layouts and make a landscape version of that

**Pull request closed without merging**